.github/workflows/release.yml

```diff
@@ -24,13 +24,9 @@ jobs:
     - name: Test with pytest
       run: |
         pytest --cov=./calibr8 --cov-append --cov-report xml --cov-report term-missing calibr8/tests.py
-    - name: Install and test with PyMC v4
-      run: |
-        pip install "pymc==4.4.0"
-        pytest --cov=./calibr8 --cov-append --cov-report xml --cov-report term-missing calibr8/tests.py
     - name: Install and test with PyMC v5
       run: |
-        pip install "pymc==5.0.0"
+        pip install "pymc>=5.0.0"
```

Comment:
This might be a bit dangerous in case PyMC has breaking changes?

Reply:
Then at least we'll notice..

Ideally we should set up a dependabot config to have it open a PR with every new PyMC release: https://github.com/michaelosthege/mcbackend/blob/main/.github/dependabot.yml

Do you want to try dependabot?